src/styles/style.css

```diff
@@ -0,0 +1,365 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.nav {
+  display: flex;
+  position: fixed;
+  height: 64px;
+  width: 100%;
+  background-color: var(--blue-100);
+  box-shadow: 2px 2px 2px #4a54b1;
+  justify-content: space-between;
+  transition: 0.5s;
+  z-index: 1;
+}
+.logo {
+  padding-left: 18px;
+  padding-top: 22px;
+}
+.nav ul {
+  display: flex;
+  padding-right: 18px;
+  list-style-type: none;
+}
+.nav ul li {
+  margin: 0px 15px;
+  line-height: 64px;
+  color: var(--blue-600);
+  text-decoration: none;
```

Comment:
You are setting `text-decoration` on the `li` element and not on the `a`, which will probably work because of how CSS works, but can introduce confusion.